src/Converter.php

```diff
@@ -1,45 +1,34 @@
 <?php
 
+declare(strict_types=1);
+
 namespace Money;
 
 /**
  * Provides a way to convert Money to Money in another Currency using an exchange rate.
- *
- * @author Frederik Bosch <f.bosch@genkgo.nl>
  */
 final class Converter
 {
-    /**
-     * @var Currencies
-     */
-    private $currencies;
+    private Currencies $currencies;
 
-    /**
-     * @var Exchange
-     */
-    private $exchange;
+    private Exchange $exchange;
 
     public function __construct(Currencies $currencies, Exchange $exchange)
     {
         $this->currencies = $currencies;
-        $this->exchange = $exchange;
+        $this->exchange   = $exchange;
     }
 
-    /**
-     * @param int $roundingMode
-     *
-     * @return Money
-     */
-    public function convert(Money $money, Currency $counterCurrency, $roundingMode = Money::ROUND_HALF_UP)
+    public function convert(Money $money, Currency $counterCurrency, int $roundingMode = Money::ROUND_HALF_UP): Money
     {
         $baseCurrency = $money->getCurrency();
-        $ratio = $this->exchange->quote($baseCurrency, $counterCurrency)->getConversionRatio();
+        $ratio        = $this->exchange->quote($baseCurrency, $counterCurrency)->getConversionRatio();
 
-        $baseCurrencySubunit = $this->currencies->subunitFor($baseCurrency);
+        $baseCurrencySubunit    = $this->currencies->subunitFor($baseCurrency);
         $counterCurrencySubunit = $this->currencies->subunitFor($counterCurrency);
-        $subunitDifference = $baseCurrencySubunit - $counterCurrencySubunit;
+        $subunitDifference      = $baseCurrencySubunit - $counterCurrencySubunit;
 
-        $ratio = (string) Number::fromFloat($ratio)->base10($subunitDifference);
+        $ratio = (string) Number::fromFloat((float) $ratio)->base10($subunitDifference);
```

Comment:
While we are at it, you already converted the return type for `CurrencyPair::getConversionRatio()` from [float](https://github.com/moneyphp/money/blob/a44e2f2bb6bf048596af1649cb7c51dd421a979a/src/CurrencyPair.php#L96) to [string](https://github.com/Ocramius/money/blob/feature/quick-money/src/CurrencyPair.php#L92), so let's also change `Number::fromFloat()` to `Number::fromString()` and get rid of the floats.

Reply:
eb32c53dbc32ad31a387735efcb2a7e6024f0a0c